### PR TITLE
Added nhif & health facilities queries

### PR DESCRIPTION
**Pull request by samoei**
This update branch includes the nhif and health facilities queries to the lambda function.

Also notice i have removed all the functions which were not being used.

### CODE REVIEW

File: lambda_function.py [hunk 1]
```diff
@@ -58,67 +62,81 @@ def build_query_response(query):
...
+        return [msg, r.json()]
     # If we miss the keywords then reply with the prefered query formats
     else:
         msg_items = []
         msg_items.append("We could not understand your request.")
         msg_items.append("Example query for doctors: DR SAMUEL AMAI")
         msg_items.append("Example query for clinical officers: CO SAMUEL AMAI")
         msg_items.append("Example query for nurse officers: NO SAMUEL AMAI")
+        msg_items.append("Example query for nhif accredited hospitals: NHIF KITALE")
+        msg_items.append("Example query for health facility: HF KITALE")
```
Comment (DavidLemayian) [inline lambda_function.py]: Let's make it:
```python
msg_items.append("Example query for health facility: Hosp. KITALE")
```

Let's actually change the entire response as:

>We could not understand your query. Try these:
>1. Doctors: DR. SAMUEL AMAI
>2. Clinical Officers: CO SAMUEL AMAI
>3. Nurses: NURSE SAMUEL AMAI
>4. NHIF accredited hospital: NHIF KITALE
>5. Health Facility: HF KITALE

Comment (DavidLemayian) [reply lambda_function.py]: @samoei ☝️

File: lambda_function.py [hunk 2]
```diff
@@ -58,67 +62,81 @@ def build_query_response(query):
...
 
 
-def construct_message(docs_list):
-    if len(docs_list) < 1:
-        return "Could not find a doctor with that name or the location you provided is currently not served by an NHIF accredited hospital"
-
+def construct_nhif_response(nhif_list):
+    # Just incase we found ourselves here with an empty list
+    if len(nhif_list) < 1:
+        return "Could not find a nurse with that name"
```
Comment (DavidLemayian) [inline lambda_function.py]: Let's fix up not being able to find this.

Comment (DavidLemayian) [reply lambda_function.py]: Message:
> The location you provided is currently not served by an NHIF accredited hospital.

File: lambda_function.py [hunk 3]
```diff
@@ -58,67 +62,81 @@ def build_query_response(query):
...
+    return "\n".join(msg_items)
 
 
-def construct_co_response(co_list):
+def construct_hf_response(hf_list):
     # Just incase we found ourselves here with an empty list
-    if len(co_list) < 1:
-        return "Could not find a clinical officer with that name"
+    if len(hf_list) < 1:
+        return "Could not find a nurse with that name"
```
Comment (DavidLemayian) [inline lambda_function.py]: Fix on the response message here as well.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
@samoei I've left a couple of changes that need to happen before accepting the merge.

File: lambda_function.py [hunk 4]
```diff
@@ -110,7 +117,7 @@ def construct_co_response(co_list):
 def construct_nhif_response(nhif_list):
     # Just incase we found ourselves here with an empty list
     if len(nhif_list) < 1:
-        return "Could not find a nurse with that name"
+        return "The location you provided is currently not served by an NHIF accredited hospital."
```
Comment (DavidLemayian) [inline lambda_function.py]: Let's change this to:
```python
return "We could not find an NHIF accredited hospital in the location you provided."
```

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
@samoei Just a small fix.

File: lambda_function.py (hunk 4 above)
Comment (samoei) [reply lambda_function.py]: @DavidLemayian I have updated this

**Review by DavidLemayian — APPROVED** [review]

**Pull request merged**